src/OpenTelemetry.Instrumentation.SqlClient/SqlClientInstrumentationOptions.cs

```diff
@@ -61,95 +64,121 @@ public class SqlClientInstrumentationOptions
-        /// <para>See also: <a href="https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Instrumentation.SqlClient#Enrich">example</a>.</para>
+        /// <para><b>Enrich is only executed on .NET and .NET Core
+        /// runtimes.</b></para>
+        /// The parameters passed to the enrich action are:
+        /// <list type="number">
+        /// <item>The <see cref="Activity"/> being enriched.</item>
+        /// <item>The name of the event. Currently only <c>"OnCustom"</c> is
+        /// used but more events may be added in the future.</item>
+        /// <item>The raw <c>SqlCommand</c> object from which additional
+        /// information can be extracted to enrich the <see
+        /// cref="Activity"/>.</item>
+        /// </list>
         /// </remarks>
-        /// <example>
-        /// <code>
-        /// using var tracerProvider = Sdk.CreateTracerProviderBuilder()
-        ///     .AddSqlClientInstrumentation(opt => opt.Enrich
-        ///         = (activity, eventName, rawObject) =>
-        ///      {
-        ///         if (eventName.Equals("OnCustom"))
-        ///         {
-        ///             if (rawObject is SqlCommand cmd)
-        ///             {
-        ///                 activity.SetTag("db.commandTimeout", cmd.CommandTimeout);
-        ///             }
-        ///         }
-        ///      })
-        ///     .Build();
-        /// </code>
-        /// </example>
         public Action<Activity, string, object> Enrich { get; set; }
 
-#if !NETFRAMEWORK
         /// <summary>
-        /// Gets or sets a Filter function that determines whether or not to collect telemetry about a command
-        /// The Filter gets the SqlCommand, and should return a boolean.
-        /// If Filter returns true, the request is collected.
-        /// If Filter returns false or throw exception, the request is filtered out.
+        /// Gets or sets a filter function that determines whether or not to
+        /// collect telemetry about a command.
         /// </summary>
         /// <remarks>
-        /// <para>object: the raw <c>SqlCommand</c> object to interrogate for a decision on whether to trace or not.</para>
+        /// <para><b>Filter is only executed on .NET and .NET Core
+        /// runtimes.</b></para>
+        /// Notes:
+        /// <list type="bullet">
+        /// <item>The first parameter passed to the filter function is the raw
+        /// <c>SqlCommand</c> object for the command being executed.</item>
+        /// <item>The return value for the filter function is interpreted as:
+        /// <list type="bullet">
+        /// <item>If filter returns <see langword="true" />, the command is
+        /// collected.</item>
+        /// <item>If filter returns <see langword="false" /> or throws an
+        /// exception the command is NOT collected.</item>
+        /// </list></item>
+        /// </list>
```

Comment:
Add the remark about "only supported on .NET and .NET Core" for `Filter` as well?

Reply:
It's there, Line 153 😄

Reply:
Whoops! 😂